multigeiger/userdefines-example.h

```diff
@@ -5,14 +5,16 @@
 // and then edit userdefines.h as appropriate for your use case / hardware.
 //====================================================================================================================================
 
+#ifndef CPU
```

Comment:
please add a comment that when using platformio, this is defined in platformio.ini.

Reply:
ok